LoopKit/GlucoseKit/GlucoseMath.swift

```diff
@@ -87,13 +87,16 @@ extension BidirectionalCollection where Element: GlucoseSampleValue, Index == In
     /// - Parameters:
     ///   - duration: The duration of the effects
     ///   - delta: The time differential for the returned values
-    ///   - velocityMaximum: The limit on how fast the momentum effect can rise. Defaults to 4 mg/dL/min based on physiological rates
+    ///   - velocityMaximum: The limit on how fast the momentum effect can be. Defaults to 4 mg/dL/min based on physiological rates, if nil passed.
     /// - Returns: An array of glucose effects
-    func linearMomentumEffect(
-        duration: TimeInterval = TimeInterval(minutes: 30),
-        delta: TimeInterval = TimeInterval(minutes: 5),
-        velocityMaximum: HKQuantity = HKQuantity(unit: HKUnit.milligramsPerDeciliter.unitDivided(by: .minute()), doubleValue: 4.0)
+    public func linearMomentumEffect(
+        duration: TimeInterval = TimeInterval(15 /* minutes */ * 60 /* seconds */),
+        delta: TimeInterval = TimeInterval(5 /* minutes */ * 60 /* seconds */),
+        velocityMaximum: HKQuantity? = nil
     ) -> [GlucoseEffect] {
+
+        let velocityMax = velocityMaximum ?? HKQuantity(unit: HKUnit.milligramsPerDeciliter.unitDivided(by: .minute()), doubleValue: 4.0)
+
```

Comment:
why not just default to `HKQuantity(unit: HKUnit.milligramsPerDeciliter.unitDivided(by: .minute()), doubleValue: 4.0)` instead of `nil`?

Reply:
`HKUnit.milligramsPerDeciliter` is an internal HKUnit extension and can't be used as a default argument in a public function.

Reply:
hm... then how does the existing code work? This code change is moving away from that default. Confused?

Reply:
ah... I missed `public`. Got it.